packages/components/Molecules/Search/index.js

```diff
@@ -1,7 +1,11 @@
 import RplSearchForm from './SearchForm.vue'
 import RplSearchResults from './SearchResults.vue'
+import RplSearchResultsTable from './SearchResultsTable.vue'
+import RplSearchResultsV2 from './SearchResultsLayout.vue'
```

Comment:
We may add a deprecate message in `SearchResults ` to remind users to use `SearchResultsLayout `. An example: https://github.com/dpc-sdp/ripple/blob/3358427e782379eae15f500b0a8aaa34a30169f1/packages/components/Molecules/Card/CardHonourRoll.vue#L41

Reply:
Add deprecate message